Arreglar pthread_mutex_t desconocido

Este fichero de cabecera asume que se incluyen otros ficheros de cabecera implícitamente,
pero no es el caso con la configuración actual del conjunto de herramientas
de compilación. Así pues, los incluimos explícitamente para que el compilador sepa dónde
encontrar la definición de algunos tipos de datos.

Signed-off-by: AlexTMjugador <AlexTMjugador@users.noreply.github.com>
---
diff -pur linux.orig/tools/thermal/tmon/tmon.h linux/tools/thermal/tmon/tmon.h
--- linux.orig/tools/thermal/tmon/tmon.h	2019-07-03 13:13:45.000000000 +0200
+++ linux/tools/thermal/tmon/tmon.h	2019-07-12 23:08:28.407436432 +0200
@@ -16,6 +16,8 @@
  *
  */
 
+#include <pthread.h>
+
 #ifndef TMON_H
 #define TMON_H
 
